examples/llama2/llama2_genai.json

```diff
@@ -47,7 +47,53 @@
             }
         }
     },
+    "evaluators": {
+        "common_evaluator": {

```

Comment:
Why do you need to add evaluators and enable search_strategy for this example?

Reply:
No we don't. Pushed up a bad commit.